src/modules/extendedCallWindow/i18n/nb_NO.js

```diff
@@ -0,0 +1,169 @@
+module.exports = {
+    arrCounter: {
+        resetTexts: {
+            counter: 'AAR-Counter',
+            highlight: 'AAR outline',
+            selection: 'Vehicle selection',
+            counter_highlight: 'AAR counter / outline',
+            counter_selection: 'AAR counter and vehicle selection',
+            highlight_selection: 'AAR outline and vehicle selection',
+            counter_highlight_selection:
+                'AAR counter / border and vehicle selection',
+        },
+        reset: '{text} reset',
+    },
+    arrHover: {
+        reset: 'The vehicle selection is reset before!',
+        titles: {
+            set: 'So often this combination is selected when you press the AAR',
+            attribute: 'The combination described in this line',
+            free: 'So often this combination is available',
+            max:
+                'You can select this AAR until this combination is no longer available',
+        },
+        arrSpecs: {
```

Comment:
You can add this now and do the rest at a later moment:
        fire: 'Brannbil', 
        elw: 'Innsatsleder brann', 
        dlk: 'Stigebil', 
        rtw: 'Ambulanse', 
        rw: 'Tungredningsbil', 
        elw2: 'Innsatsstøttebil', 
        elw1_or_elw2: 'Innsatsleder brann eller innsatsstøttebil', 
        hlf_only: 'Redningsbil', 
        hlf_or_rw_and_lf: 'Redningsbil eller Tungredningsbil og Brannbil', 
        gwa: 'Røykdykkerbil', 
        gwl2wasser: 'Tankbil', 
        gwgefahrgut: 'CBRNe-enhet', 
        fustw: 'Patruljebil', 
        police_motorcycle: 'Politimotorsykkel', 
        fustw_or_police_motorcycle: 'Patruljebil eller Politimotorsykkel', 
        polizeihubschrauber: 'Politihelikopter', 
        swat: 'Politi med våpentrening', 
        swat_armored_vehicle: 'Pansret kjøretøy', 
        swat_suv: 'Delta kjøretøy', 
        any_traffic_car: 'Utrykningspolitibil', 
        k9: 'Hundepatrulje', 
        rth_only: 'Luftambulanse', 
        gw_taucher: 'Dykkerbil', 
        boot: 'Båter (generelt)', 
        mzb: 'Lett redningsbåt',